examples/integrations/oci/oci-secrets-mp-config-source/README.md

```diff
@@ -0,0 +1,331 @@
+# OCI Secrets MP Config Source Example
+
+This example project demonstrates the use of the `integrations/oci/oci-secrets-mp-config-source` library.
+
+## Prerequisites
+
+* Java 21+
+* Maven 3.8+
+* An [Oracle Cloud Infrastructure (OCI) account](https://www.oracle.com/cloud/).
+
+## Overview
+
+This example shows how to configure and use a Microprofile Config configuration source, via Helidon's Microprofile
+Config meta-configuration feature, that can transparently fetch text values from an OCI vault resource. It exercises the
+classes present in the `integrations/oci/oci-secrets-mp-config-source` project elsewhere in this source code repository.
+
+## One-Time Setup Steps
+
+Because this example makes use of Oracle Cloud, you will need to set up some Oracle Cloud resources before you
+begin. You should have to perform these steps only once.
+
+### Create OCI Resources and Gather OCI Information
+
+0. Open a text document on your computer where you can save some important information related to OCI resources as you
+   go along. This example will presume this document is located in `/tmp/helidon-examples-ocids.txt`.
+
+1. Create a new, empty `${HOME}/.oci/config` file whose contents you will fill in shortly. This example presumes that
+   you do not already have one. If you do have one, you may need to edit it carefully rather than following these
+   instructions.
+
+    1. (There are many ways to authenticate to your Oracle Cloud account that are supported by Helidon. This is just one
```

Comment:
Please provide a link to the documentation or README for the other authentication configuration methods.

Reply:
The relative URL to the README should be fine.